readme.md

```diff
@@ -45,10 +45,33 @@ Type: `object`
 
 ##### deep
 
-Type: `boolean`<br>
+Type: `boolean | Function`<br>
 Default: `false`
 
-Recurse nested objects and objects in arrays.
+Recursing behaviour for nested objects and objects in arrays
+
+Deep Value can be boolean:
+
+- If `true`, it recurses all nested objects and objects in arrays.
+
+Or a `DeepKeyFilter` function which has the following properties:
```

Comment:
`DeepKeyFilter` means nothing in JS as the type is not available there.

Reply:
ok, I agree.